README.md

```diff
@@ -442,12 +442,16 @@ A curated list of awesome Java frameworks, libraries and software.
 
 *Everything else.*
 
+* [Avian](https://github.com/ReadyTalk/avian) - Open source JVM with both a JIT & AOT modes. Includes an iOS port.
+* [Codename One](https://www.codenameone.com/) - Open source cross platform solution for writing native mobile apps in Java for iOS, Android, Windows etc.
```

Comment:
Please be as concise as possible, i.e.

`Cross platform solution for writing native mobile apps.`